plenoptic/simulate/models/frontend.py

```diff
@@ -47,63 +41,95 @@ class Front_End(nn.Module):
         parasol on   fine time coarse space
         parasol off
     """
-    def __init__(self, disk_mask=False):
-        super().__init__()
-
-        filts = np.load(dirname + '/front_end_filters.npy')
-        scals = np.load(dirname + '/front_end_scalars.npy')
-
-        linear = nn.Conv2d(in_channels=1, out_channels=2, kernel_size=31, bias=False)
-        linear.weight = nn.Parameter(torch.tensor((filts[0], filts[3]),
-                                     dtype=torch.float32).unsqueeze(1))
-
-        luminance = nn.Conv2d(in_channels=1, out_channels=2, kernel_size=31,
-                              bias=False)
-        luminance.weight = nn.Parameter(torch.tensor((filts[1], filts[4]),
-                                        dtype=torch.float32).unsqueeze(1))
 
-        contrast = nn.Conv2d(in_channels=2, out_channels=2, kernel_size=31,
-                             groups=2, bias=False)
-        contrast.weight = nn.Parameter(torch.tensor((filts[2], filts[5]),
-                                       dtype=torch.float32).unsqueeze(1))
-
-        pad = nn.ReflectionPad2d(filts.shape[-1]//2)
-
-        self.luminance_scals = torch.tensor((scals[0], scals[2]), dtype=torch.float32)
-        self.luminance_scals = self.luminance_scals.view((1, 2, 1, 1))
+    def __init__(self, disk_mask=False, pretrained=False, requires_grad=True):
+        """
+        Parameters
+        ----------
+        disk_mask: bool
+            Apply circular Gaussian mask to center of image. The mask itself is square.
+        pretrained: bool
+            Load weights from Berardino et al. 2017. These are 31x31 convolutional filters.
+        requires_grad: bool
+            Whether or not model is trainable.
+        """
+        super().__init__()
```

Comment:
convolutional filters are always 31x31 now, right? even if `pretrained=False`. can we update docstring to make that clear?

Reply:
done.